src/components/CohortBuilder/ParticipantsTableView/SaveSetModal.tsx

```diff
@@ -111,7 +107,7 @@ const SaveSetModal: FunctionComponent<Props> = (props) => {
   const maxNumOfCharsToDisplay = MAX_LENGTH_NAME + 1;
 
   useEffect(() => {
-    if (error) {
+    if (error && !isSaveSetNameConflictError(error)) {
```

Comment:
I think the first check if unnecessary: error && !isSaveSetNameConflictError(error) => !isSaveSetNameConflictError(error)

But make sure it's ok